Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputViewModel.swift

```diff
@@ -123,6 +144,8 @@ class CreditCardInputViewModel: ObservableObject {
         }
     }
 
+    // MARK: Business logic
```

Comment:
We should add tests for business logic

Reply:
@lmarceau, this should already have tests (CreditCardInputFieldTests). I only added the `MARK:` for readability. 

Let me know if you think we can add any unit test here that we might be missing?

Reply:
`CreditCardInputFieldTests` is testing `CreditCardInputField`. I was referring more to make sure we test the `CreditCardInputViewModel` with that comment. I see we have a `CreditCardInputViewModelTests`, so let's just make sure the current changes are covered in that class

Reply:
`CreditCardInputViewModel` coverage is 51.3% which is slightly higher than our current limit (50%). I see some func like `udpateCreditCard` and the new `dismissAndRemoveCreditCard` that have not unit test and it will be good to add some

Reply:
Hold on, I found something to test ha

Reply:
Thank you both, the more tests the better. @yoanarios I added test for `update` and `removal` both.